app/config/parameters.yml.dist

```diff
@@ -68,3 +68,7 @@ parameters:
     jira_issue_type: spd-delete-production-entity
     jira_issue_entityid_fieldname: customfield_13018
     jira_issue_project_key: CXT
+
+    # Playground uri's for OIDC entities
+    playground_uri_test: https://test.dev.playground.surfconext.nl
```

Comment:
These should also end up in the Ansible parameters template.

Reply:
Thanks!